homeassistant/components/unifiprotect/const.py

```diff
@@ -41,11 +45,24 @@
 MIN_REQUIRED_PROTECT_V = Version("1.20.0")
 OUTDATED_LOG_MESSAGE = "You are running v%s of UniFi Protect. Minimum required version is v%s. Please upgrade UniFi Protect and then retry"
 
+SERVICE_SET_DOORBELL_MESSAGE = "set_doorbell_message"
+
+TYPE_EMPTY_VALUE = ""
+
 PLATFORMS = [
     Platform.BUTTON,
     Platform.CAMERA,
     Platform.LIGHT,
     Platform.MEDIA_PLAYER,
     Platform.NUMBER,
+    Platform.SELECT,
     Platform.SWITCH,
 ]
+
+SET_DOORBELL_LCD_MESSAGE_SCHEMA = vol.Schema(
```

Comment:
If this service schema is only used in the select platform, please move it to that module. Having the schema in another module just makes it harder to read the platform code. This goes for all constants.